README.md

```diff
@@ -2,44 +2,145 @@
 
 ## Usage
-Models can be composed of mixins for various estimators and forecast processes.
+Models can be composed of mixins for various estimators and forecast processes. These composable
+pieces can be put together in different ways to solve many problems. The RegularTimeSeriesModel is the base.
+The core that problem specific parts are added to when forecasting or gap filling a particular timeseries.
+The estimator is the next essential building block. It can be a Classifier or a Regressor. There are many 
+different numerical techniques that can be applied via the estimator. The process is the last essential 
+component. It defines the timeseries being forecast and the available feature data that might have predictive value.
+Having composed a Model class from these three parts, it is then up to the user to create an instance of the class that 
+with configuration arguments that tune the model features for the specific meter load or pv forecast. 
+
 
-An ordinary least squares model for a Balancing Area Hourly Load
+<!-- Explain what this does - for lawyers... point to ipynb example -->
+<!-- Consider adding sklearn model diagram if it would help (rather than hurt) the explanation -->
+
+### Compose a model class
+
+A model using an Ordinary Least Squares estimator to forecast a Balancing Area Hourly Load
 ```python
 class NpOlsModel(BalancingAreaHourly, LinearRegressor, RegularTimeSeriesModel):
   pass
 ```
-An XGBoost model for Balancing Area Hourly Load
+
+A model using an XgBoost estimator to forecast AMI (meter) Hourly Load
 ```python
-class XgbModel(BalancingAreaHourly, XgbRegressor, RegularTimeSeriesModel):
+class XgbModel(AmiHourlyForecast, XgbRegressor, RegularTimeSeriesModel):
   pass
 ```
 
-The initialization arguments control feature construction.
+Additional behaviors like filters can also be added via composition. See the notebook demo for examples.
 
+### Configure a model instance
+The initialization arguments control feature construction. The configuration arguments are specific to the
+process and the estimator used to compose the model.
 
-### Trained models should be fully serializable using Cloud Pickle for maximum flexibility
+As an example, we can configure XgbModel from above with three types of features: lagged correlations, one hot encoded 
+day-of-week values and a harmonic feature that decomposes time into a sine and cosine waves with the specified frequencies.
+```python
+config = dict(
+lags = np.array([24, 48, 168], dtype="timedelta64[h]"),
+day_of_week=True,
+harmonics = np.array([24, 168, 365*24], dtype="timedelta64[h]")
+)
+instance = XgbModel(**config)
+```
 
+### Fit & Predict
+Once a model instance is created, we must train the machine learning algorithm. The fit method takes
+a start date, stop date and a list of one or more identifiers. For process classes that support it, this
+allows training a single model for a cohort of resources.
```

Comment:
Do some process classes simply not allow any learning? If not, why would they not support training?

P3